composer.json

```diff
@@ -1,33 +1,35 @@
 {
-    "name": "php-pm/php-pm",
```

Comment:
y

On Aug 30, 2016 18:53, "Marc J. Schmidt" <notifications@github.com> wrote:

> In composer.json
> <https://github.com/php-pm/php-pm/pull/164#discussion_r76834684>:
>
> > @@ -1,33 +1,35 @@
> >  {
> > -    "name": "php-pm/php-pm",
>
> Can you please fix it?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/php-pm/php-pm/pull/164/files/0c45bf827cfa3bc7e902125ca8a8536b9ddc656a#r76834684>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABqUD3sJiV-CoG9mw8xrv3FwbZYJtmknks5qlF_pgaJpZM4JwdWg>
> .
>